src/passes/Print.cpp

```diff
@@ -2582,7 +2582,9 @@ struct PrintSExpression : public UnifiedExpressionVisitor<PrintSExpression> {
 
   void handleSignature(HeapType curr, Name name = Name()) {
     Signature sig = curr.getSignature();
-    if (!name.is() && getTypeSystem() == TypeSystem::Nominal) {
+    bool nominal = !name.is() && (getTypeSystem() == TypeSystem::Nominal ||
```

Comment:
`nominal` seems like a confusing name here. That is, `foo = isFoo() || isBar()` just feels odd...

How about `notStructural`, if nothing better?

Reply:
Hmm, I guess I can use `hasSupertype`.